cmd/validate.go

```diff
@@ -0,0 +1,80 @@
+// Copyright © 2023
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/equinor/radix-operator/pkg/apis/radixvalidators"
+	"github.com/equinor/radix-operator/pkg/apis/utils"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v3"
+)
+
+// logoutCmd represents the logout command
+var validateCmd = &cobra.Command{
+	Use:   "validate",
+	Short: "Validate radixconfig.yaml",
+	Long:  `Check radixconfig.yaml for structural and logical errors`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+
+		cmd.SilenceUsage = true
+
+		radixconfig, err := cmd.Flags().GetString("radixconfig")
```

Comment:
```suggestion
		radixconfig, err := cmd.Flags().GetString("config-file")
```
fix incorrect argument name